src/toil/wdl/wdltoil.py

```diff
@@ -624,26 +610,37 @@ def unpack_toil_uri(toil_uri: str) -> Tuple[FileID, str, str]:
 
     return file_id, parent_id, file_basename
 
-def evaluate_output_decls(output_decls: List[WDL.Tree.Decl], all_bindings: WDL.Env.Bindings[WDL.Value.Base], standard_library: ToilWDLStdLibBase) -> WDL.Env.Bindings[WDL.Value.Base]:
+def evaluate_bindings_from_decls(decls: List[WDL.Tree.Decl], all_bindings: WDL.Env.Bindings[WDL.Value.Base], standard_library: ToilWDLStdLibBase,
```

Comment:
Maybe this should be `evaluate_decls_to_bindings`, or `generate_bindings_from_decls`, or even just `evaluate_decls`? I think the decls are what gets evaluated, not the bindings.